lib/systems/examples.nix

```diff
@@ -11,36 +11,31 @@ rec {
 
   sheevaplug = rec {
     config = "armv5tel-unknown-linux-gnueabi";
-    arch = "armv5tel";
     float = "soft";
     platform = platforms.sheevaplug;
   };
 
   raspberryPi = rec {
     config = "armv6l-unknown-linux-gnueabihf";
-    arch = "armv6l";
     float = "hard";
     fpu = "vfp";
     platform = platforms.raspberrypi;
   };
 
   armv7l-hf-multiplatform = rec {
-    config = "arm-unknown-linux-gnueabihf";
-    arch = "armv7-a";
+    config = "armv7a-unknown-linux-gnueabihf";
```

Comment:
This change doesn' seem related and how do you know this doesn't break something?

Reply:
It is related in that all the `parsed.cpu.name` stuff would get `arm` instead of `armv7-a`. This is a different string but at least means the same thing. If those build systems just pass `--arch` right to `gcc` with `-march` or whatever, then we should have them use `hostPlatform.platform.gcc.arch` instead of `hostPlatform.parsed.cpu.name`.